Gemfile.lock

```diff
@@ -1,238 +0,0 @@
-GEM
```

Comment:
Can you give me more details on the problems arised with Gemfile.lock? It is important to have it on version control to make sure when we install the dependencies to run locally we are all on the same page. A reference on this: https://bundler.io/v1.12/bundler_workflow.html

Reply:
@filipemcarvalho `nokogiri` dependency installation froze everything I ran `bundle install`. I searched about it but only found unrelated similar problems dating back to 2014-2015.
Maybe you can merge this PR, and then add the `Gemfile.lock` yourself afterwards?